stockflux-watchlist/src/components/ClosedWindows/ClosedWindows.js

```diff
@@ -1,7 +1,7 @@
-import React, { useState } from "react";
-import moment from "moment";
-import closedWindowsImageInactive from "../../assets/png/closed_tabs.png";
-import closedWindowsImageActive from "../../assets/png/closed_tabs_active.png";
+import React, { useState } from 'react';
+import moment from 'moment';
+import closedWindowsImageInactive from 'stockflux-components/src/images/closed_tabs.png';
```

Comment:
We should use icon classes for these rather than importing them directly from source of stockflux-components

Reply:
Fixed -> 08cff915e0c7c162c3c71c72e9adb0e02ba1de6c